_posts/2018-02-28-Update-to-Java11.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: post
+author: arto
+title: Update to Java 11 now!
+excerpt: What's coming up in world of JVM
+tags:
+- Java
+- Java 11
+- Software development
+---
+
+I recently wrote an article on updating your environment to Java 9 for better modularity and less memory and disk footprint. Turns out, that was not cutting edge enough, since many are already using Java 9 to run production systems. Also, after my article, there was some news coverage on how Java 9 and 10 will end up having very short support period, and many people potentially jumping straight to Java 11. Let's do that!
```

Comment:
~and less memory and disk footprint~ and reduced memory and disk use

"Turns out that was not cutting edge enough since…" (no commas)

"having *a* very short"